src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -183,6 +187,39 @@ class ProgressPane
                 return;
             }
 
+            if (ExperimentalFeature.IsEnabled("PSAnsiProgress"))
+            {
+                Console.CursorVisible = false;
```

Comment:
Should we restore this?

Reply:
yes, had it elsewhere but ended up removing that code and not putting back the restoration